checkov/cloudformation/graph_builder/graph_components/blocks.py

```diff
@@ -36,9 +36,9 @@ def __init__(
     def update_attribute(
         self, attribute_key: str,
         attribute_value: Any,
-        change_origin_id: int,
+        change_origin_id: int | None,
```

Comment:
using `|` was introduced in Python 3.10, I think. And is the preferred way to use instead of `Optional`, but at the end they both do the same. You just don't need to import it 🙂